pkgs/applications/office/jameica/default.nix

```diff
@@ -76,6 +77,13 @@ stdenv.mkDerivation rec {
     install -Dm644 build/jameica-icon.png $out/share/pixmaps/jameica.png
     cp ${desktopItem}/share/applications/* $out/share/applications/
 
+    # Create .app bundle for macOS
```

Comment:
Can you end the multiline string after the cp ${desktopItem} line, and then concatenate a lib.optionalString with the darwin-specific bits? This seems more consistent with the rest of nixpkgs, especially when conditionally adding other multiline strings.

Reply:
thanks!